Sources/Transformer.swift

```diff
@@ -6,44 +6,20 @@
 //  Copyright © 2016 Syo Ikeda. All rights reserved.
 //
 
-public struct Transformer<From, To> {
-    private let transform: From throws -> To
+public struct Transformer<F, T>: TransformerType {
+    public typealias From = F
+    public typealias To = T
+    private let transform: From -> To?
 
-    public init(_ transform: From throws -> To) {
+    public init(_ transform: From -> To?) {
         self.transform = transform
     }
 
     /// - Throws: DecodeError
```

Comment:
It depends on the `transform` block, so change `transform` type from `From throws -> To` to `From -> To?`.